### ISSUE TITLE
Error getting your geolocation, using IP location

### ISSUE THREAD

**Issue by Mr10001** [iam4x/pokemongo-webspoof#329]
The webspoof app used to work prior to the recent Pokemon Go update and after as well. It seems, (I'm unsure if this is related or not) as if after setting up: https://github.com/AHAAAAAAA/PokemonGo-Map 

I receive the following errors and the map never loads within the webspoof app. I have done this on another OSX machine that is up to date as well and encounter the same exact issue now after setting up the PokemonGo-Map and using it. One of the requirements is to setup your own google API key. Again, I'm unsure if this is related or not.

Thank you for any help with this devs!

Errors:
**Error getting your geolocation, using IP location**
"Network location provider at 'https://www.googleapis.com/' : Returned error code 403."

![image](https://cloud.githubusercontent.com/assets/13803395/17335515/db7f0eb6-589f-11e6-8329-e1ffa01ea25c.png)

**Could not use IP location**
Try to restart app, report issue to github

"TypeError: Cannot read property 'split' of undefined"
![image](https://cloud.githubusercontent.com/assets/13803395/17335531/e81ccb0e-589f-11e6-95e6-1065f11c6837.png)

This is similar to:
https://github.com/iam4x/pokemongo-webspoof/issues/259
https://github.com/iam4x/pokemongo-webspoof/issues/237
https://github.com/iam4x/pokemongo-webspoof/issues/192

**Comment by nickg819** [comment]
Why create a new ticket if you know it's related to a bunch of other ones?

**Comment by Mr10001** [comment]
After reviewing "index.js" that I found when "viewing package contents" of the .app file , I noticed:

  // geolocation API might be down, use http://ipinfo.io
  // source: http://stackoverflow.com/a/32338735

and that if the geolocation call fails, then it tries:

http://ipinfo.io/

Turns out that ipinfo.io was being blocked via DNS on the network that I was on.

The fix was, use a VPN, a different network, or if you have access turn off the blocking of those sites.

PS- Anyone know what the URL/IP that is used to make the original API call?
